airflow-core/tests/unit/serialization/test_dag_serialization.py

```diff
@@ -1229,10 +1234,6 @@ def test_extra_serialized_field_and_operator_links(
             link = simple_task.get_extra_links(ti, name)
             assert link == expected
 
-        # Test Deserialized link registered via Airflow Plugin
-        link = simple_task.get_extra_links(ti, GoogleLink.name)
-        assert link == "https://www.google.com"
```

Comment:
We would need to test this part as well right?

Reply:
I will take a look tomorrow - possibly I cna solve the issue differently